operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go

```diff
@@ -41,6 +44,10 @@ type MultiClusterObservabilitySpec struct {
 	ObservabilityAddonSpec *observabilityshared.ObservabilityAddonSpec `json:"observabilityAddonSpec"`
 }
 
+// T Shirt size class for a particular o11y resource.
+// +kubebuilder:validation:Enum:={"default","minimal","small","medium","large","xlarge","2xlarge","4xlarge","8xlarge"}
```

Comment:
based on comments in the doc yesterday, were we going to remove the 8xlarge?

Reply:
Yup, can remove, was planning on doing it last so people have time to comment